src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1663,12 +1700,13 @@ private void visitConstructorFunctionDeclaration(FunctionType ftype) {
     }
 
     private void visitFunctionParameters(FunctionType ftype) {
-      visitFunctionParameters(ftype, true);
+      visitFunctionParameters(ftype, true, Sets.<String>newHashSet());
```

Comment:
whats the difference between `Collections.<String>emptySet()` and `Sets.<String>newHashSet()`? I am no java expert, but use one of the two consistently.